style.scss

```diff
@@ -0,0 +1,388 @@
+$mainFontSize:1rem;
```

Comment:
Nebála bych se tady odřádkovat/odkomentovat, které proměnné se týkají textu, které barev, které velikosti mezer aj. Ve velkých projektech bys na to měla i samostatný soubor, čistě pro čitelnost. :sunflower: